literature.bib

```diff
@@ -13751,27 +13815,27 @@ @article{BB08
 }
 
 @inproceedings{HKW:ICSE08,
-  author = {Florian Heidenreich and Jan Kopcsek and Christian Wende},
-  title = {{FeatureMapper: Mapping Features to Models}},
-  booktitle = ICSEC,
-  pages = {943--944},
-  location = {Leipzig, Germany},
+	author = {Florian Heidenreich and Jan Kopcsek and Christian Wende},
+	title = {{FeatureMapper: Mapping Features to Models}},
+	booktitle = ICSEC,
+	pages = {943--944},
+	location = {Leipzig, Germany},
 	publisher = ACM,
 	address = NY,
-  note = {Informal demonstration paper},
+	note = {Informal demonstration paper},
 	tt-tags = {Feature Modeling Tool,SPL Implementation Tool},
-  month = MAY,
-  year = 2008
+	month = MAY,
+	year = 2008
 }
 
 @inproceedings{SSP08,
```

Comment:
SS:SPLC08?

Reply:
Good catch. This is actually not a reference from my PR but I will fix this. This might break some \cite in other papers though. I'll add a note on that renaming.

Reply:
I now realise that we actually have a lot of these bugs (directly after this reference).
ACPPW08 -> ACP+:POPL08
KTA08 -> KTA:ViSPLE08
KAK08 -> KAK:ICSE08
MWCC08 -> MWCC:GPCE08
KA08 -> KA:ASE08

How many ill-formed keys do we have actually?
As these "bugs" are not related to this PR, I won't change anything about this yet.
Still we should discuss if we want to fix this or keep the keys with consistency for old papers.
I believe that it is very unlikely for duplicate keys to occur with these old references.
What do you think @tthuem ?

Reply:
After discussion with Thomas we agreed on fixing this but documenting such renamings.